applications/ShapeOptimizationApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -167,6 +167,8 @@ void  AddCustomUtilitiesToPython(pybind11::module& m)
         .def(py::init<ModelPart&>())
         .def("ComputeUnitSurfaceNormals", &GeometryUtilities::ComputeUnitSurfaceNormals)
         .def("ProjectNodalVariableOnUnitSurfaceNormals", &GeometryUtilities::ProjectNodalVariableOnUnitSurfaceNormals)
+        .def("ProjectNodalVariableToDirection", &GeometryUtilities::ProjectNodalVariableToDirection)
```

Comment:
```suggestion
        .def("ProjectNodalVariableOnDirection", &GeometryUtilities::ProjectNodalVariableOnDirection)
```

I think "on" is the proper word here. Also, the other functions need to be adjusted accordingly.